mozjs/src/typedarray.rs

```diff
@@ -252,8 +252,10 @@ impl<T: TypedArrayElementCreator + TypedArrayElement, S: JSObjectStorage> TypedA
     }
 
     ///  Update an existed JS typed array
-    pub unsafe fn update(&mut self, data: &[T::Element]) {
-        Self::update_raw(data, self.object.as_raw());
+    pub fn update(&mut self, data: &[T::Element]) {
+        unsafe {
+            Self::update_raw(data, self.object.as_raw());
+        }
```

Comment:
@gterzian do you think we can merge this one?

Reply:
Yes I think so. @jdm ?

Reply:
Yeah, I don't see any way to.misyse this interface.